pygls/protocol.py

```diff
@@ -378,6 +379,11 @@ def _send_data(self, data):
         if not data:
             return
 
+        if IS_PYODIDE:
+            from js import post_message
```

Comment:
This requires there be a JavaScript `post_message` method be defined on the server's web worker.
See [here](https://github.com/openlawlibrary/pygls/blob/f6d1f9e3947a9175b5998424db3da9ebcbdefc22/examples/fountain-extension/src/server.ts#L34-L41)